core/models/asset.py

```diff
@@ -257,7 +257,7 @@ class Asset(models.Model):
                                      null=True,
                                      on_delete=models.PROTECT)
     current_status = models.CharField(editable=False, max_length=50)
-    asset_condition = models.CharField(editable=False,
+    condition_notes = models.CharField(editable=False,
```

Comment:
while we're here.. can we remove this default value?

Reply:
also, let's use `asset_condition_notes` all through

Reply:
@nelmandela To ensure that notes on the condition of an asset are not limited to 50 chars, prefer TextField over CharField.